src/DiagnosticSourceDemo/CorrelationVectorPropagation.AspNetClassic/CorrelationVectorPropagation.AspNetClassic.csproj

```diff
@@ -0,0 +1,61 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{4B2ABFC2-3237-48AA-B938-A87B1DCD2CC4}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>CorrelationVectorPropagation.AspNetClassic</RootNamespace>
+    <AssemblyName>CorrelationVectorPropagation.AspNetClassic</AssemblyName>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```

Comment:
>v4.6.1 [](start = 28, length = 6)

Do we want to target a framework version below 4.6.1 to see if the events emitted are different?

Reply:
Switched to 4.6, but can't go lower than that because I'm using AsyncLocal. If a goal is to support earlier versions of .NET, we can work around that but will leave at 4.6 for now.